PyTado/http/http.py

```diff
@@ -83,6 +83,14 @@ class Http:
     # Id
     id = 0
 
+    # Request Header Information
+    request_headers = {
+        'Content-Type': 'application/json',
+        'Origin': 'https://app.tado.com',
+        'Referer': 'https://app.tado.com/',
+        'User-Agent' : 'PyTado'
```

Comment:
This seems like easy to block as well. Do we know what the official app uses?

Reply:
usually, its browser :( we can hack it in.. maybe make that dynamic at some point.